src/host/main.cpp

```diff
@@ -298,6 +295,17 @@ int main(int argc, char** argv)
       "defined by this timer interval.")
     ->capture_default_str();
 
+  size_t client_connection_timeout = 1000;
```

Comment:
1 second seems very aggressive compared to the default (exponential up to 64s)?

Reply:
As discussed, this is now 2s so we get at least one retry from the OS, while keeping this value below the default raft election timeout.